packages/editor/src/plugins/image/components/licence-dropdown.tsx

```diff
@@ -0,0 +1,111 @@
+  const [isOpen, setIsOpen] = useState(false)
+  const [triggerWidth, setTriggerWidth] = useState(0)
+
+  const triggerRef = useRef<HTMLLabelElement>(null)
+
+  useEffect(() => {
+    if (isPixabayImage) {
+      const pixabayLicence =
+        licenceNames.find((name) => name.toLowerCase().includes('pixabay')) ||
+        ''
+      setSelectedLicense(pixabayLicence)
+      onLicenseChange?.(pixabayLicence)
+      setIsConfirmed(true)
+    }
+  }, [isPixabayImage, onLicenseChange, licenceNames])
+
+  useEffect(() => {
+    setTriggerWidth(triggerRef.current ? triggerRef.current.offsetWidth / 3 : 0)
+  }, [])
+
+  const handleLicenseChange = (newLicence: string) => {
+    setSelectedLicense(newLicence)
+    onLicenseChange?.(newLicence)
+    setIsConfirmed(true)
+  }
+
+  return (
+    <label
+      ref={triggerRef}
+      className="mx-auto mb-0 mt-5 flex items-center text-almost-black"
+    >
+      <span className="serlo-tooltip-trigger w-1/4">
+        Lizenz
+        <FaIcon className="ml-2" icon={faQuestionCircle} />
+        <EditorTooltip text={licenceHelpText} />
+      </span>
+      <span className="flex w-3/4 flex-row border-2 border-transparent">
+        <Select.Root
+          onValueChange={handleLicenseChange}
+          defaultValue={selectedLicense}
+          onOpenChange={setIsOpen}
+          value={selectedLicense}
+        >
+          <Select.Trigger
+            className={cn(
+              'serlo-input-font-reset rounded-md border-2 border-editor-primary-100 bg-editor-primary-100 px-2 text-left focus:border-2 focus:border-editor-primary',
+              'flex flex-row justify-between',
+              isOpen &&
+                'mb-[-1] rounded-b-none border-b-0 border-editor-primary-300'
+            )}
+            style={{ width: `${triggerWidth}px` }}
+          >
+            <Select.Value placeholder="Select an option" />
+            <Select.Icon className="ml-2 text-almost-black">
+              <FaIcon icon={isOpen ? faCaretUp : faCaretDown} />
+            </Select.Icon>
+          </Select.Trigger>
+          <Select.Content
+            className="w-full rounded-md rounded-t-none border-2 border-t-0 border-editor-primary bg-white"
+            style={{ width: `${triggerWidth}px` }}
+            side="bottom"
+            sideOffset={0}
+            position="popper"
+          >
+            {licenceNames.map((license) => (
+              <Select.Item
+                key={license}
+                value={license}
+                className="serlo-input-font-reset my-0 px-3 hover:bg-editor-primary-100"
+              >
+                <Select.ItemText>{license}</Select.ItemText>
+              </Select.Item>
+            ))}
+          </Select.Content>
+        </Select.Root>
+        {isConfirmed && (
+          <span className="pointer-events-none ml-2 inline flex items-center pr-3 text-green-500">
+            &#10003;
+          </span>
+        )}
```

Comment:
I don't understand this tick tbh. Maybe more an UX Problem but if I want it to be CC BY-SA 4.0 I will never get a tick because I did not change it, correct?

Reply:
Correct - I just pushed the correct behaviour for the default value.

From the issue:

> Per Default wird die Standart-Lizenz CC BY-SA 4.0 ausgewählt
> 
> Sobald eine Änderung an den Lizenzen vorgenommen wurde: Grünes Häkchen setzen
> 
> für spätere Iteration (bei Lust jetzt schon einbauen): Falls keine Änderung an der Lizenz vorgenommen wird noch einmal via Popup beim User nachfragen, ob das Bild unter CC BY-SA 4.0 steht. Dann automatisch grünes Häkchen setzen

Reply:
for @GregorZupan: As a user testing this feature I was mostly unsure what that tick meant and was slightly confused.